packages/tessellate-request/lib/configurations/token-storage.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+
+import manageTokens from 'node-tokens'
+import * as configurationProvider from './configuration-provider'
+import { OAUTH2_JSON_FILE_BACKED_USER_CREDENTIALS_PROVIDER } from './configuration-type'
+
+const GET_TOKEN_RETRY_COUNT = 3
+const GET_TOKEN_INTERVAL = 500 //ms
+
+let managedTokens;
+
+function startManagingTokens() {
+  managedTokens = {}
+  const configurations: Array<AuthConfig> = configurationProvider.getByType(OAUTH2_JSON_FILE_BACKED_USER_CREDENTIALS_PROVIDER)
+
+  configurations.forEach(configuration => {
```

Comment:
I changed the `forEach()` to `for... of`. In this case, I do not see a benefit of using a generator function (the array will most likely contain one or two small objects).